stdlib/test/tempfile/test_tempfile.mojo

```diff
@@ -177,7 +177,59 @@ fn test_temporary_directory() raises -> None:
     assert_false(exists(tmp_dir), "Failed to delete temp dir " + tmp_dir)
 
 
+fn test_named_temporary_file_deletion() raises:
```

Comment:
Here and elsewhere in the file: could you please use `def` instead of `fn` + `raises`? We're trying to close the gap between these and prefer `def` when possible.